src/tests/Login.test.jsx

```diff
@@ -107,3 +108,23 @@ it('handles wrong credentials', async () => {
     
     expect(screen.getByLabelText('errorMessage')).toHaveTextContent("The server is currently unavailable. Try again later")
   })
+
+it('Checking if the fields are empty', async () => {
```

Comment:
It's best to start the test name in its present tense instead of continuous form as the test using the format `it ......`, so using `it + checks ...` feels more appropriate then `it + checking ... 

Note that the test name doesn't start in capital letter either.